addons/website/static/tests/core/interaction.test.js

```diff
@@ -1662,8 +1676,11 @@ describe("throttledForAnimation", () => {
         class Test extends Interaction {
             static selector = ".test";
             dynamicContent = {
-                "_root:t-on-click": this.throttledForAnimation(this.doSomething),
+                "_root:t-on-click": () => this.throttle(),
```

Comment:
No idea why I can't do `"t-on-click": this.throttle` directly, but it doesn't work. :thinking: